x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/index.ts

```diff
@@ -5,9 +5,4 @@
  * 2.0.
  */
 
```

Comment:
renaming the directory made the git diff a lot better since i did not reuse a lot of code